frontend/src/Editor/CodeBuilder/CodeHinter.jsx

```diff
@@ -33,8 +33,9 @@ import { camelCase } from 'lodash';
 import { useTranslation } from 'react-i18next';
 import cx from 'classnames';
 import { Alert } from '@/_ui/Alert/Alert';
-import { useCurrentState } from '@/_stores/currentStateStore';
+import { useCurrentStateStore } from '@/_stores/currentStateStore';
```

Comment:
Can we import only the state instead of complete store? It might expose the setting currentState actions as well